### DISCUSSION TITLE
Missing syntax snapshot

### DISCUSSION THREAD

**Discussion by danielpetrov01** [JSQLParser/JSqlParser#2038] [Q&A] [answered]
Hello, there used to be a supported syntax tree for all the different statements.
They used to be located here:
https://jsqlparser.github.io/JSqlParser/syntax_snapshot.html

Now there is nothing located there.

**Comment by manticore-projects** [comment, marked as answer]
Thank you for your report. We struggle with Snapshot deployment and will fix it.
In the meantime, you can find the Grammar Rail Road Diagram here: https://manticore-projects.com/JSQLParser/syntax_stable.html

**Reply by danielpetrov01** [reply]
Thank you very much!
